zvariant/src/dict.rs

```diff
@@ -151,6 +154,14 @@ impl<'k, 'v> Dict<'k, 'v> {
         }
     }
 
+    pub fn iter(&self) -> impl Iterator<Item = (&'_ Value<'k>, &'_ Value<'v>)> {
```

Comment:
Are the explicit lifetime elisions (`'_`) really needed?

Reply:
Tested locally, not required and removed.